index.html

```diff
@@ -20,20 +20,192 @@
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+    </div>
+    </div>
+    <div id="section-threeb">
+        <div class="lap">
+        <img src="./asset/laptop2image.png" alt="">
+    </div>
+    </div>
+    <div class="section-threeb">
+        <div class="main-se">
+        <div class="one"><p>My Project Portfolio</p></div>
+        <div class="two"><button type="button">HTML/CSS</button></div>
+        <div class="three"><button type="button">Ruby on Rails</button></div>
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+    </div>
     </div>
+<div id="section-threec">
+    <div class="lap">
+    <img src="./asset/laptopfour.png" alt="">
+</div>
+</div>
+<div class="section-threeb">
+    <div class="main-se">
+    <div class="one"><p>My Project Portfolio</p></div>
+    <div class="two"><button type="button">HTML/CSS</button></div>
+    <div class="three"><button type="button">Ruby on Rails</button></div>
+    <div class="four"><button type="button">Javascript</button></div>
+    <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+</div>
+</div>
+<div id="section-threed">
+    <div class="lap">
+    <img src="./asset/laptopfive.png" alt="">
+</div>
+</div>
+<div class="section-threeb">
+    <div class="main-se">
+    <div class="one"><p>My Project Portfolio</p></div>
+    <div class="two"><button type="button">HTML/CSS</button></div>
+    <div class="three"><button type="button">Ruby on Rails</button></div>
+    <div class="four"><button type="button">Javascript</button></div>
+    <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+</div>
+</div>
+<div id="section-threee">
+    <div class="lap">
+    <img src="./asset/laptopsix.png" alt="">
+</div>
+</div>
+<div class="section-threeb">
+    <div class="main-se">
+    <div class="one"><p>My Project Portfolio</p></div>
+    <div class="two"><button type="button">HTML/CSS</button></div>
+    <div class="three"><button type="button">Ruby on Rails</button></div>
+    <div class="four"><button type="button">Javascript</button></div>
+    <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+</div>
+</div>
+<div class="About">
+    <div class="My">
+    <h2>About</h2>
+    <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+    <button type="button">Get my CV <img src="./asset/Rectangledown.png" alt="u-shape" ><img src="./asset/downarrow.png" alt="" class="imgarrow"></button>
+</div>
+</div>
```

Comment:
- [ ] The about section should be wrapped in a `section` tag for reasons I mentioned in a previous comment 🏄